### PR TITLE
Kcho/link dpanonymize

### PR DESCRIPTION
**Pull request by kcho**
@tashrifbillah This PR goes with https://github.com/AMP-SCZ/dpanonymize. Currently, `dpanonymize` copies files from the PROTECTED to GENERAL. Once this placeholder is tested to be intact, we can work on `dpanonymize` further to remove all PII and link with other PII removing tools for video, audio, actigraphy and etc.

Thanks!

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -19,3 +19,4 @@ paramiko = "*"
 pytz = "*"
 requests = {extras = ["security"]}
 lochness = {path = ".", editable = true}
+bpanonymize = "*"
```
Comment (tashrifbillah) [inline Pipfile:22]: The word is `dp...`, why is it `bp...`?

File: setup.py [hunk 2]
```diff
@@ -14,7 +14,8 @@
     'six',
     'pytz',
     'pandas',
-    'jsonpath_ng'
+    'jsonpath_ng',
+    'bpanonymize @ git+https://kcho@github.com/AMP-SCZ/dpanonymize@v0.1#egg=bpanonymize'
```
Comment (tashrifbillah) [inline setup.py:18]: I inserted missing `v` [here](https://github.com/AMP-SCZ/dpanonymize/releases/tag/v0.1)

Comment (tashrifbillah) [reply setup.py:18]: I only know about this syntax:

> pip install git+https://github.com/aladagemre/django-notification.git@v0.1

Have you tested Ln 18 as written?

File: Pipfile (hunk 1 above)
Comment (kcho) [reply Pipfile:22]: Thanks for spotting this. It's typo. I'll fix all of your comments once your review is done.

File: setup.py [hunk 3]
```diff
@@ -15,7 +15,7 @@
     'pytz',
     'pandas',
     'jsonpath_ng',
-    'bpanonymize @ git+ssh://kcho@github.com/AMP-SCZ/dpanonymize@v0.1#egg=bpanonymize'
+    'bpanonymize@git+https://github.com/AMP-SCZ/dpanonymize@v0.1#egg=bpanonymize'
```
Comment (tashrifbillah) [inline setup.py:18]: Yeah this is expected to work since I have made the repo public.

Comment (kcho) [reply setup.py]: Now `dpanonymize` uploaded to pypi. I'll need to fix here and there for the `dpanonymize`, but at least the installation process for `dpanonymize` work fine linked in `lochness` installation.

File: scripts/sync.py [hunk 4]
```diff
@@ -155,6 +156,9 @@ def do(args):
             for Module in args.source:
                 lochness.attempt(Module.sync, Lochness, subject, dry=args.dry)
 
+    # annonymize PII
+    dpanonymize.lock_lochness(Lochness)
```
Comment (tashrifbillah) [inline scripts/sync.py:160]: `dpanonymize.anon_lochness` can be a better name :)

Comment (kcho) [reply scripts/sync.py:160]: `lock` sounds fine to me. If you insist to go with anon_lochness, I can change.

**Review by tashrifbillah — APPROVED** [review]
Tashrif has reviewed all except the tests and does not have any more suggestions.

**Pull request merged**